src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,48 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_SIZE = 10;
+    private Object[] key;
+    private Object[] value;
+    private int sizeTracker = 0;
+
+    public StorageImpl() {
+        this.key = new Object[ARRAY_SIZE];
+        this.value = new Object[ARRAY_SIZE];
+    }
 
     @Override
     public void put(K key, V value) {
+        if (getIndex(key) != -1) {
+            this.value[getIndex(key)] = value;
+        } else {
+            this.key[sizeTracker] = key;
+            this.value[sizeTracker] = value;
+            sizeTracker++;
+        }
     }
 
     @Override
     public V get(K key) {
-        return null;
+        return getIndex(key) != -1 ? ((V) value[getIndex(key)]) : null;
     }
 
     @Override
     public int size() {
+        return sizeTracker;
+    }
+
+    private int getIndex(K key) {
+        for (int i = 0; i < sizeTracker; i++) {
+            if (key == null && this.key[i] == null) {
```

Comment:
```suggestion
            if (key == keys[i]) {
```
keys will be either identical non-null, or both null